packages/bpk-component-popover/src/BpkPopover.js

```diff
@@ -40,7 +42,19 @@ const bindEventSource = (source, callback) => event => {
   callback(event, { source });
 };
 
-const BpkPopover = props => {
+export type Props = {
+  id: string,
+  onClose: Function,
+  label: string,
+  closeButtonText: string,
+  children: Node,
+  className: ?string,
```

Comment:
Can we put maybe types last? Makes it easier to read

Reply:
Sure